src/ocean/util/serialize/contiguous/Util.d

```diff
@@ -65,11 +66,25 @@ public Contiguous!(S) copy(S) ( Contiguous!(S) src, ref Contiguous!(S) dst )
 
 *******************************************************************************/
 
-public Contiguous!(S) copy(S) ( ref S src, ref Contiguous!(S) dst )
+version (D_Version2) // because `in ref` is not legal D1
```

Comment:
`in ref` == `const ref`
Could it be `(ref Const!(S) src, ref Contiguous!(S) dst)` ?